src/components/MobileNavbar/MobileNavbar.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import styles from './MobileNavbar.module.css';
+import newMeeting from '../../assets/images/my-meetings.svg';
+import perfil from '../../assets/images/perfil.svg';
+import myMeetings from '../../assets/images/my-meetings.svg';
+
+function MobileNavbar() {
+    return(
+        <div className={ styles["footer"] }>
```

Comment:
```suggestion
        <footer className={ styles["footer"] }>
```
Se podría usar la etiqueta `footer`

Reply:
👍